pkgs/applications/networking/cluster/kuma/default.nix

```diff
@@ -1,19 +1,39 @@
-{ lib, fetchFromGitHub, buildGoModule }:
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, coredns
+, isFull ? true
+, enableGateway ? false
+, pname ? "kuma"
+, version ? "1.3.1"
+, components ? ["kumactl"] ++ lib.optionals isFull [
```

Comment:
Put the ++ and the rest of the line on the next line

Reply:
Thank's for review, i just changed it.